client-tools/Dump/DumpFeature.cpp

```diff
@@ -640,9 +640,8 @@ DumpFeature::DumpFeature(Server& server, int& exitCode)
   using arangodb::basics::FileUtils::buildFilename;
   using arangodb::basics::FileUtils::currentDirectory;
   _options.outputPath = buildFilename(currentDirectory().result(), "dump");
-  _options.threadCount =
-      std::max(uint32_t(_options.threadCount),
-               static_cast<uint32_t>(NumberOfCores::getValue()));
+  // Default ThreadCount 4
+  _options.threadCount = std::max(uint32_t(_options.threadCount), 4);
```

Comment:
A simpler solution would be to remove this code here and simply adjust the default for `Options._threadCount` (from currently 2) to 4 in `client-tools/Dump/DumpFeature.h`.